languages/ruby/exercises/concept/instance-variables/.meta/design.md

```diff
@@ -0,0 +1,38 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the concept instance variables and "nil" in Ruby.
+
+## Learning objectives
+
+- Know that objects can store state using instance variables
+- Know that instance variables should be treated as private and to use getter and setter methods
+- Know that `nil` is a Ruby object used to represent "nothingness"
+- Know that instance variables are `nil` before they are assigned otherwise
+
+## Out of scope
+
+- Boolean values (True/False)
+- Truthy/Falsey-ness
+
+## Concepts
+
+- `instance-variables`
+- `nil`
+
+## Prerequisites
+
+- `basics`
```

Comment:
Basics is also not a code thing.

```suggestion
- Basics
```

Reply:
This is a token which this exercise depends on to be unlocked (by the basics exercise), not a specific code thing